tasks/jira.py

```diff
@@ -0,0 +1,74 @@
+import os
+import re
+import sys
+
+try:
+    from atlassian import Jira
+except ImportError:
+    pass
+from invoke import task
+
+from tasks.libs.notify.jira_failing_tests import get_failing_tests_names
+
+
+def get_jira():
+    username = os.environ['ATLASSIAN_USERNAME']
+    password = os.environ['ATLASSIAN_PASSWORD']
+    jira = Jira(url="https://datadoghq.atlassian.net", username=username, password=password)
+
+    return jira
+
+
+def close_issue(jira, issue_key: str, verbose_test: str):
+    print('Marking as done issue', issue_key, 'for test', verbose_test)
+
+    jira.issue_add_comment(issue_key, 'Marking this issue as done since the test is not failing anymore')
+    jira.issue_transition(issue_key, 'Wont Do')
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Given previous thread:
```suggestion
    jira.issue_transition(issue_key, 'Done')
```
But at the end of the day I can live with `Won't do` too